enginetest/script_queries.go

```diff
@@ -2006,6 +2006,20 @@ var ScriptTests = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "createing UUID quickly is still random",
```

Comment:
Rather than making this a script test, just do a normal query test

`select uuid() = uuid()` should be false